packages/aws-cdk-lib/custom-resources/README.md

```diff
@@ -824,3 +824,88 @@ new cr.AwsCustomResource(this, 'CrossAccount', {
   })]),
 });
 ```
+
+#### Custom Resource Config
+
+You can configure every CDK-vended custom resource in a given scope with `CustomResourceConfig`. 
+
+*Please understand that there is no guarantee the order in which Custom Resource Config is run. More than one Custom Resource Config of the same scope might result in conflict.
+
+The following example configures every custom resource in this CDK app to retain its logs for ten years:
+```ts
+    import { CustomResourceConfig } from 'aws-cdk-lib/custom-resources';
+
+    CustomResourceConfig.of(app).addLogRetentionLifetime(logs.RetentionDays.TEN_YEARS);
+
+    const stack = new cdk.Stack(app);
+
+    let websiteBucket = new s3.Bucket(stack, 'WebsiteBucket', {});
+    new s3deploy.BucketDeployment(stack, 's3deployNone', {
+      sources: [s3deploy.Source.jsonData('file.json', { a: 'b' })],
+      destinationBucket: websiteBucket,
+    });
+```
+The following example configures every custom resource in a nested stack to retain its log for ten years:
+```ts
+export class RootStack extends cdk.Stack {
```

Comment:
Can you avoid making new classes for these? This example never shows these classes being instantiated. You can save a bunch of space by doing:

```
const app = new App();
const stack = new Stack(app, 'Stack');
const nestedStack = new NestedStack(stack, 'NestedStack');

new s3deploy.BucketDeployment(...);

CustomResourceConfig.of(app).addLogRetentionLifetime(...);
```